daemon/logs.go

```diff
@@ -18,9 +18,16 @@ import (
 	"github.com/docker/docker/pkg/stdcopy"
 )
 
-// ContainerLogs hooks up a container's stdout and stderr streams
-// configured with the given struct.
-func (daemon *Daemon) ContainerLogs(ctx context.Context, containerName string, config *backend.ContainerLogsConfig, started chan struct{}) error {
+// ContainerLogsRawChan, given a channel, returns raw log message structs from
+// the container logger.
+func (daemon *Daemon) ContainerLogsRawChan(ctx context.Context, containerName string, config *backend.ContainerLogsConfig, msgs chan *logger.Message) error {
```

Comment:
But the logging format is API-specific, right? I'm not sure the backend is the right place to encode it in this format.